### DISCUSSION TITLE
^0.4.0 is unusable for me.

### DISCUSSION THREAD

**Discussion by vlwkaos** [vscode-neovim/vscode-neovim#1381] [General]
Something is off with visual mode since 0.4.0.
Actual cursor and what is shown on vscode constantly desyncs.
I am experiencing almost all the recent issues regarding visual mode.

**Comment by xiyaowong** [comment]
use version0.3.2

**Reply by vlwkaos** [reply]
I am using version 0.3.2, but vscode keeps updating it to the latest 🤨

**Reply by JackCA** [reply]
I can confirm that 0.3.2 also fixes this for me

**Comment by theol0403** [comment]
Please provide more information. Is it caused by config/setting? Is it actually a desync? Note that the vscode cursor has to be used for the selection and a fake cursor is used for the actual cursor pos.

**Reply by vlwkaos** [reply]
The most notable bug for me is when I move cursor position from one line to another (even adjacent ones by moving through j,k) then entering insert mode. 

It will enter insert mode in the last cursor position not at the cursor position that I am seeing. 

So it seems like actual cursor position falls one history behind the fake cursor.

To add to a possible cause, my code base includes non-english characters (that is unicode) heavily.

**Reply by JackCA** [reply]
I have this same issue

**Comment by theol0403** [comment]
Please be more specific. Have you tried with empty init.vim? Can you provide reproduction steps?
